requirements.txt

```
@@ -1,10 +1,10 @@
 black==22.3.0
-flake8==4.0.1
+flake8==5.0.4
 flake8-tidy-imports==4.6.0
-graviton@git+https://github.com/algorand/graviton@v0.3.0
+graviton@git+https://github.com/algorand/graviton@4227d96c68a2aa8793923fa5d9e982eed709f599
```

```suggestion
graviton@git+https://github.com/algorand/graviton@v0.5.0
```

... after graviton is merged and tagged